code/src/components/Game.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+import endPageImage from 'images/end-game-image.jpg'
+import soundFile from 'images/angels-singing.mp3'
+import { Direction } from './Direction';
+
+export const Game = () => {
+  const description = useSelector((store) => store.game.description);
+  const [showEndPage, setShowEndPage] = useState(false);
+  const onRestartButtonClick = () => {
+    window.location.reload()
+  }
+
+  useEffect(() => {
+    const audio = new Audio(soundFile); // Create an Audio object with the sound file
+    if (showEndPage) {
+      audio.play(); // Play the sound when showEndPage becomes true
+    }
+  }, [showEndPage]);
+
+  const ResetBtn = styled.button`
```

Comment:
Maybe create a separate file for your design and styled components to keep function-files a bit cleaner/easy to follow :) But just an input, nothing major!